pkg/apis/hive/v1alpha1/validating-webhooks/clusterdeployment_validating_admission_hook.go

```diff
@@ -285,6 +297,15 @@ func (a *ClusterDeploymentValidatingAdmissionHook) validateCreate(admissionSpec
 		if gcp.Region == "" {
 			allErrs = append(allErrs, field.Required(gcpPath.Child("region"), "must specify GCP region"))
 		}
+		// TODO: Uncomment once ready for GCP
```

Comment:
Should uncomment now. We don't need to wait for GCP for this validation to be here.

Reply:
Oh, duh. The fields are not in the machine pool platform, yet.